test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/CustomSearchParamTests.cs

```diff
@@ -39,85 +39,88 @@ public CustomSearchParamTests(HttpIntegrationTestFixture fixture, ITestOutputHel
-            {
-                Skip.If(!_fixture.IsUsingInProcTestServer, "Reindex is not enabled on this server.");
-                return;
-            }
 
-            await WaitForReindexStatus(reindexJobUri, "Running", "Completed");
+                await WaitForReindexStatus(reindexJobUri, "Running", "Completed");
 
-            FhirResponse<Parameters> reindexJobResult = await Client.CheckReindexAsync(reindexJobUri);
-            Parameters.ParameterComponent param = reindexJobResult.Resource.Parameter.FirstOrDefault(p => p.Name == "searchParams");
+                FhirResponse<Parameters> reindexJobResult = await Client.CheckReindexAsync(reindexJobUri);
+                Parameters.ParameterComponent param = reindexJobResult.Resource.Parameter.FirstOrDefault(p => p.Name == "searchParams");
 
-            Assert.Contains("http://hl7.org/fhir/SearchParameter/Patient-foo", param.Value.ToString());
+                Assert.Contains(searchParamPosted.Resource.Url, param.Value.ToString());
 
-            reindexJobResult = await WaitForReindexStatus(reindexJobUri, "Completed");
-            _output.WriteLine("Reindex job is completed, it should have reindexed the Patient resources with foo");
+                reindexJobResult = await WaitForReindexStatus(reindexJobUri, "Completed");
+                _output.WriteLine($"Reindex job is completed, it should have reindexed the resources with {randomName}");
 
-            var floatParse = float.TryParse(
-                reindexJobResult.Resource.Parameter.FirstOrDefault(predicate => predicate.Name == "resourcesSuccessfullyReindexed").Value.ToString(),
-                out float resourcesReindexed);
+                var floatParse = float.TryParse(
+                    reindexJobResult.Resource.Parameter.FirstOrDefault(predicate => predicate.Name == "resourcesSuccessfullyReindexed").Value.ToString(),
+                    out float resourcesReindexed);
 
-            _output.WriteLine($"Reindex job is completed, {resourcesReindexed} Patient ressources Reindexed");
+                _output.WriteLine($"Reindex job is completed, {resourcesReindexed} resources Reindexed");
 
-            Assert.True(floatParse);
-            Assert.True(resourcesReindexed > 0.0);
+                Assert.True(floatParse);
+                Assert.True(resourcesReindexed > 0.0);
 
-            // When job complete, search for resources using new parameter
-            await ExecuteAndValidateBundle($"Patient?{searchParam.Code}:exact={patientName}", expectedPatient.Resource);
-
-            // Clean up new SearchParameter
-            await DeleteSearchParameterAndVerify(searchParamPosted.Resource);
+                // When job complete, search for resources using new parameter
+                await ExecuteAndValidateBundle($"Appointment?{searchParam.Code}=noshow&_tag={tag.Code}", expectedAppointment.Resource);
```

Comment:
When reading through this test, I found myself searching for the "noshow" part of this string. Nit: maybe we could make `status="noshow"` a variable and use it when we are creating the Appointment resource that we are filtering for?

Reply:
Updated to use the resource status value.